pyproject.toml

```diff
@@ -21,17 +21,23 @@ dependencies = [
     "uvicorn",
     "ga4gh.vrs[extras]~=2.0.0a11",
     "sqlalchemy~=1.4.54",
+    "pyaml",
```

Comment:
out of curiosity, any reason for `pyaml` and not `pyyaml`?

Reply:
Probably the code snippet I used to import logging config used `pyaml`.  But it appears that `pyyaml` would be sufficient.  So that can be changed.